docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,34 +1,57 @@
 [[connect-to-elasticsearch]]
 == Add data to {kib}
 
-To start working with your data in {kib}, you can:
+The fastet way to add data to {kib} is to use one of the built-in options,
+available from the home page. You can:
 
-* Upload a CSV, JSON, or log file with the File Data Visualizer.
+* Add data from popular apps and services
+* Add data using an Elastic Agent
+* Upload a file
 
-* Upload geospatial data with the GeoJSON Upload feature.
+[role="screenshot"]
+image::images/add-data-home.png[Built-in options for adding data to Kibana]
 
-* Index logs, metrics, events, or application data by setting up a Beats module.
+If you're not ready to use your own data, you can add a <<get-started, sample data set>>
+to give {kib} a test drive.
 
-* Connect {kib} with existing {es} indices.
+[float]
+[[add-data-tutorial-kibana]]
+===  Add data tutorials
 
-If you're not ready to use your own data, you can add a <<get-started, sample data set>>
-to see all that you can do in {kib}.
+These tutorials guide you through installing and configuring a
+Beats data shipper to collect metrics, logs, security events, and application data and send it to {es}.
+You can then use the pre-built dashboards to explore and analyze the data.
+
+If a tutorial doesn’t exist for your data, go to the {beats-ref}/beats-reference.html[Beats overview]
+to learn about other data shippers in the Beats family.
+
+[role="screenshot"]
+image::images/add-data-tutorials.png[Add Data tutorials]
+
+
+[discrete]
+[[add-data-fleet-kibana]]
+===  Add an Elatic Agent
+
+beta[]
+
+Fleet is a single, unified way to collect logs and metrics from systems
```

Comment:
This description blurs the lines between Fleet and agent. Agent is how you collect logs/metrics. Fleet is the UI you use to add and manage agents. You might want to look at the Fleet overview (thought honestly I want to rework that because it's a bit disjointed). https://www.elastic.co/guide/en/ingest-management/7.10/fleet-overview.html